.changeset/brave-otters-switch.md

```diff
@@ -0,0 +1,5 @@
+---
+'@itwin/itwinui-react': minor
+---
+
```

Comment:
missing: jsdocs, changesets, documentation page

Reply:
Added jsdocs, changeset, will work on docs page.

Reply:
current changeset is too much 😅 it needs to be concise enough to not take more than a few lines

i think the composition example can be removed, and the descriptions for all the subcomponents can also be removed